package.json

```diff
@@ -8,6 +8,10 @@
     ".": {
       "require": "./lib.cjs/index.cjs",
       "import": "./lib.esm/index.mjs"
+    },
+    "./bundler-utils": {
```

Comment:
I don't quite like this, but this is common functionality needed for `strato-rollup-plugin` (and bundlers in general)

This export is only for internal consumption.